stable/ambassador/values.yaml

```diff
@@ -83,7 +83,7 @@ service:
   #  Uncomment out fields you want to use.
   #  See https://www.getambassador.io/reference/core/ambassador and
   #  https://www.getambassador.io/reference/core/tls for more info
-  #  getambassador.io/config: |
+    getambassador.io/config: |
```

Comment:
why does this need to be set?

Reply:
I got confused by it since the comment says "Uncomment the fields you want to use" it makes it seem like uncommenting out the single field is enough to get it to work. I figured since it confused me, other users were probably getting confused as well. Leaving this part uncommented would help clarify how the user should interact with this section.

Happy to take some feedback on this 🙂

Reply:
I guess it doesn't hurt 🤷‍♂ 😄 But you would still need to add in several lines (ie apiVersion kind etc) whenever adding in a config.

Reply:
Yeah it is not a perfect solution just something I thought would help. Maybe I'll add and additional comment to make this clearer.

FWIW, we would like everyone to be using CRDs at this point.